### DISCUSSION TITLE
Archinquisitor Nikodemus

### DISCUSSION THREAD

**Discussion by TomekPL-1** [secretcowlevel/DoomtrooperPublic#2] [Card Balance]
Archinquisitor Nikodemus (BROTHERHOOD) [R]
UNIQUE, INQUISITOR.
When Nikodemus enter your SQUAD and every time when your INQUISITOR wounds or kills a warrior, reveal top card from Opponent’s draw pile. If card is other than Dark Legion, Cybertronic or UNIQUE you can add to your hand a copy of it as brotherhood token. Until end of turn you can use this TOKEN and if you don't it is annihilated.
F: 8 S: 8 A: 8 V: 10

**Comment by Kite80** [comment]
I'd change the activation upon entering the field, the rest it's ok for me

**Comment by Sol-au-fein** [comment]
When Nicodemus cast an art, draw 1 card.

**Comment by Sol-au-fein** [comment]
Archinquisitor Nikodemus
F: 5 S: 7 A: 8 V: 9
UNIQUE INQUISITOR
When Nicodemus enters the game, all opponent's warriors leave cover and may not seek cover.
Art casted by Nikodemus costs one action less.

Nikodemus is arguably the most powerful Art caster, that Brotherhood can deploy. I think his card should reflect this. Now about stats, why he is weaker than Inquisitor Majoris. His tabletop fighting stats are pretty average. Certainly not equal to other combat focused heroes. He is not primarily warrior, his role is artslinging and he is pretty good at it.

`Archinquisitor Salvatore Nikodemus was born the youngest son of a commoner Bauhaus family. As youngster he showed signs of power in the Art. His family, having strong faith in the Brotherhood, promtly turned over this promising young wielder of the Aspects. This is where he assumed his current name Nikidemus (after an ancient Brotherhood martyr) under the Brotherhood's strict tutelage. Nikodemus studied hard and rose through their ranks, becoming known for his power and efficiency.
Eventually Nikodemus's skill grew into something more than useful. He was immediately promoted into the Second Directorate, namely the cell of Inquisitors. After the appropriate training he proved himself to be an efficient inquisitor. Salvatore was relenless in his search for any sign of the Dark Symmetry. This determination and drive for the light has earned him the title of Archinquisitor. The Brotherhood trusts and relies on Nikodemus to such a degree they now send only him to most difficult assignments. 
During his career Salvatore has instituted many reforms and ideas that will hold true for hundreds of years. Among these are the Death Angel, a battle car designed specifically for inquisitors. 
Nikodemus carries an artifact of deadly design based on AC-40. This weapon was carried into battle by other great Inquisitor Majores. Its name is Gabriel, and was personal armament of the first Inquisitor. It posseses intergrated flamethrower in addition to the chain bayonet.
Salvatore's graying hair accents his blue eyes, and his stern look. He is seldom ever seen without his battledress, and never appears on the field without Gabriel.`
